default.nix

```diff
@@ -3,7 +3,22 @@
 with pkgs;
 
 let
-  sources = nix-gitignore.gitignoreSource [ "default.nix" "ci" "tests" "target" ] ./.;
+  # TODO: fix gitignore and replace this by gitignre
+  # See https://github.com/NixOS/nixpkgs/issues/69138
+  sources = builtins.path {
+    name = "hydra-cli-filtered-source";
+    path = ./.;
+    filter = (path: type:
+      baseNameOf path != ".git" &&
+      baseNameOf path != "default.nix" &&
+      baseNameOf path != "ci" &&
+      baseNameOf path != "tests" &&
+      baseNameOf path != "target" &&
+      baseNameOf path != "result" &&
+      (! (pkgs.lib.hasSuffix ".rs.bk" path)) &&
```

Comment:
Can we drop those two? We cannot filter out *everything* that could be in this directory. Otherwise we also have to add `.bak` and `.tmp` and `.swp` and `.old` and .....

Reply:
This is what currently is in our `.gitignore` file :) 
We can change that, but I think it has to be in another PR.

Reply:
Hm, why is that? This PR changes how our sources are filtered and it is introducing these exclusions again- can’t we just not introduce them again? ;)

I’m not going to fight over it of course. Just surprised.